community/detector/rce/cve202135464/src/main/java/com/google/tsunami/plugins/detectors/rce/cve202135464/Cve202135464Detector.java

```diff
@@ -0,0 +1,150 @@
+        .addAllDetectionReports(
+            matchedServices.stream()
+                .filter(NetworkServiceUtils::isWebService)
+                .filter(this::isServiceVulnerable)
+                .map(networkService -> buildDetectionReport(targetInfo, networkService))
+                .collect(toImmutableList()))
+        .build();
+  }
+
+  private boolean isServiceVulnerable(NetworkService networkService) {
+    String targetUri = NetworkServiceUtils.buildWebApplicationRootUrl(networkService)
+        + QUERY_STRING;
+
+    try {
+      HttpResponse httpResponse =
+          httpClient.send(
+              get(targetUri)
+                  .withEmptyHeaders()
+                  .build(),
+              networkService);
+
+      if (httpResponse.status().code() != 200) {
+        return false;
+      }
+      //TODO: also check for content-length to be 970
+      return (httpResponse.status().code() == 200) &&
+         (httpResponse.headers().get("Content-Length").get() == "970");
+      //logger.atInfo().log("httpResponse: %s", httpResponse);
+      //return (httpResponse.status().code() == 200);
+    } catch (IOException e) {
+      logger.atWarning().withCause(e).log("Request to target %s failed", networkService);
+      return false;
+    }
+  }
+
+  private DetectionReport buildDetectionReport(
+      TargetInfo targetInfo, NetworkService vulnerableNetworkService) {
+    return DetectionReport.newBuilder()
+        .setTargetInfo(targetInfo)
+        .setNetworkService(vulnerableNetworkService)
+        .setDetectionTimestamp(Timestamps.fromMillis(Instant.now(utcClock).toEpochMilli()))
+        .setDetectionStatus(DetectionStatus.VULNERABILITY_VERIFIED)
+        .setVulnerability(
+            Vulnerability.newBuilder()
+                .setMainId(
+                    VulnerabilityId.newBuilder()
+                        .setPublisher("TSUNAMI_COMMUNITY")
+                        .setValue("CVE_2021_35464"))
+                .setSeverity(Severity.CRITICAL)
+                .setTitle("Pre-auth RCE in OpenAM 14.6.3/ForgeRock AM 7.0 (CVE-2021-35464)")
+                .setDescription(
+                    "OpenAM server before 14.6.3 and ForgeRock AM server before 7.0 have "
+                        + "a Java deserialization vulnerability in the jato.pageSession "
+                        + "parameter on multiple pages. The exploitation does not require "
+                        + "authentication, and remote code execution can be triggered by "
+                        + "sending a single crafted /ccversion/* request to the server. "
+                        + "The vulnerability exists due to the usage of Sun ONE Application "    
+                        + "Framework (JATO) found in versions of Java 8 or earlier. The issue "
+                        + "was fixed in commit a267913b97002228c2df45f849151e9c373bc47f from "
+                        + "OpenIdentityPlatform/OpenAM:master."))
+        .build();
```

Comment:
Can you also populate the `recommendation` field to provide instructions on how to resolve this vulnerability?